x-pack/plugins/observability/public/components/app/section/alerts/index.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup } from '@elastic/eui';
+import { EuiLink } from '@elastic/eui';
+import { EuiBadge } from '@elastic/eui';
+import { EuiFlexItem } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import { EuiIcon } from '@elastic/eui';
+import moment from 'moment';
+import { EuiHorizontalRule } from '@elastic/eui';
+import { SectionContainer } from '..';
+
+const alerts = [
+  {
+    id: '1',
+    name: 'Error rate | opbeans-java',
+    alertTypeId: 'apm.error_rate',
+    tags: ['apm', 'service.name:opbeans-java'],
+    updatedAt: '2020-07-03T14:27:51.488Z',
+    muteAll: true,
+  },
+  {
+    id: '2',
+    name: 'Error rate | opbeans-java',
+    alertTypeId: 'apm.error_rate',
+    tags: ['apm', 'service.name:opbeans-java'],
+    updatedAt: '2020-07-02T14:27:51.488Z',
+    muteAll: true,
+  },
+  {
+    id: '3',
+    name: 'Error rate | opbeans-java',
+    alertTypeId: 'apm.error_rate',
+    tags: ['apm', 'service.name:opbeans-java'],
+    updatedAt: '2020-06-25T14:27:51.488Z',
+    muteAll: true,
+  },
+  {
+    id: '4',
+    name: 'Error rate | opbeans-java',
+    alertTypeId: 'apm.error_rate',
+    tags: ['apm', 'service.name:opbeans-java'],
+    updatedAt: '2020-03-25T14:27:51.488Z',
+    muteAll: true,
+  },
+];
+
+export const AlertsSection = () => {
+  return (
+    <SectionContainer
+      title="Alerts"
+      subtitle={i18n.translate('xpack.observability.overview.alerts.subtitle', {
+        defaultMessage: 'Recent activity',
+      })}
+      appLink={'/app/management/insightsAndAlerting/triggersActions/alerts'}
```

Comment:
This is going to be slightly annoying, but I think in this case we want the label to not be "View in app" but rather "Manage alerts". Is it possible we can add a `appLinkLabel` to set a different label for specific sections?